app/javascript/dashboard/routes/dashboard/settings/inbox/channels/Website.vue

```diff
@@ -182,3 +210,12 @@ export default {
   },
 };
 </script>
+
+<style scoped>
+.greetings--richtext {
+  padding: 0 var(--space-normal);
+  border-radius: var(--border-radius-normal);
+  border: 1px solid #e0e6ed;
```

Comment:
Please use `var(--color-border)`

Reply:
Replaced hardcoded values with variables in [commit](https://github.com/chatwoot/chatwoot/pull/2952/commits/7a347b6a7cbab3e20b00c82f9c466aeb0d05b533)